ci: add test-build

liweijie0812 opened the pull request:
### 🤔 这个 PR 的性质是？
> 勾选规则:
> 1.只要有新增参数，就勾选”新特性提交“
> 2.只修改内部bug，未新增参数，才勾选”日常 bug 修复“
> 3.其他选项视具体改动判断

- [ ] 日常 bug 修复
- [ ] 新特性提交
- [ ] 文档改进
- [ ] 演示代码改进
- [ ] 组件样式/交互改进
- [x] CI/CD 改进
- [ ] 重构
- [ ] 代码风格优化
- [ ] 测试用例
- [ ] 分支合并
- [ ] 其他

### 🔗 相关 Issue

<!--
1. 描述相关需求的来源，如相关的 issue 讨论链接。
-->

### 💡 需求背景和解决方案

<!--
1. 要解决的具体问题。
2. 列出最终的 API 实现和用法。
3. 涉及UI/交互变动需要有截图或 GIF。
-->

### 📝 更新日志

<!--
从用户角度描述具体变化，以及可能的 breaking change 和其他风险。
-->

- fix(组件名称): 处理问题或特性描述 ...

- [x] 本条 PR 不需要纳入 Changelog

### ☑️ 请求合并前的自查清单

⚠️ 请自检并全部**勾选全部选项**。⚠️

- [ ] pr目标分支为develop分支，请勿直接往main分支合并
- [ ] 标题格式为：`组件类名`: 修改描述（示例：`TDBottomTabBar`: 修复iconText模式，底部溢出2.5像素）
- [ ] ”相关issue“处带上修复的issue链接
- [ ] 相关文档已补充或无须补充

liweijie0812 committed:
创建 test-build.yml

tencent-adm commented:
[![CLA assistant check](https://cla.opensource.tencent.com/pull/badge/signed)](https://cla.opensource.tencent.com/Tencent/tdesign-flutter?pullRequest=814) <br/>All committers have signed the CLA.

liweijie0812 committed:
Merge branch 'develop' into liweijie0812-patch-1

liweijie0812 committed:
更新 test-build.yml

liweijie0812 committed:
更新 test-build.yml

liweijie0812 committed:
更新 test-build.yml

liweijie0812 committed:
Generate key.properties for Android build

Add step to generate key.properties for Flutter build.

liweijie0812 committed:
Update test-build.yml

liweijie0812 committed:
Update test-build.yml

liweijie0812 committed:
Update test-build.yml

liweijie0812 committed:
Change Flutter setup action to use 'main' branch

Updated Flutter setup action to use 'main' branch.

liweijie0812 committed:
Modify test-build workflow triggers for develop branch

Updated the GitHub Actions workflow to trigger on push to the develop branch and removed main branch from pull request triggers.

liweijie0812 committed:
Update test-build.yml

liweijie0812 committed:
Update test-build.yml

liweijie0812 committed:
d

liweijie0812 committed:
update  matrix

liweijie0812 committed:
update  matrix
